docs/paas-plus/dev/onboarding-to-spryker-code-upgrader/how-to-connect-spryker-code-upgrader.md

```diff
@@ -0,0 +1,144 @@
+---
+title: How to connect to Spryker Code Upgrader
+description: Know how to easily connect your repository to preconfigured Spryker Code Upgrader
+template: concept-topic-template
+---
+
+The connection of your project to Spryker Code Upgrader in Spryker CI is slightly different depending on your Git hosting provider.
+
+Please follow the instructions for your Git hosting provider for the best experience.
+
+## GitHub
+
+There are 2 options for connecting your GitHub repository to Spryker Upgrade Service in Spryker CI: with help of BuddyCI application or with the access token. You can select the option that is the most suitable for you.
+
+### Integration through BuddyCI application
```

Comment:
This is easier than using the token, right?

Reply:
This is the UI way to connect the repository - with the use of the application (https://docs.github.com/en/apps/creating-github-apps/creating-github-apps/about-apps). Probably for the user, it can be easier, yes.